packages/ui-extensions/src/surfaces/checkout/api/standard/standard.ts

```diff
@@ -62,7 +62,7 @@ export interface Extension<Target extends ExtensionTarget = ExtensionTarget> {
   /**
    * The API version that was set in the extension config file.
    *
-   * @example '2024-04', '2024-07', '2024-10', '2025-01', 'unstable'
+   * @example '2024-07', '2024-10', '2025-01', '2025-04', 'unstable'
```

Comment:
I'm not sure if we want to add this yet. It's not a valid version yet so could be a little misleading. Usually we add this days before publishing.